### PR TITLE
FPL-88: Add HMCTS admin user

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPL-88

### Change description ###

Added hmcts admin user to ccd configuration. This user currently cannot complete any actions within a case and can only see submitted cases in the work basket.

```
USER DETAILS
id: hmcts-admin@example.com
password: Password12
```

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField.json [hunk 1]
```diff
@@ -229,5 +229,236 @@
...
+  },
+  {
+    "LiveFrom": 42736,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "CaseFieldID": "factorsParenting",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": 42737,
```
Comment (geek-soft) [inline ccd-definition/AuthorisationCaseField.json]: Let's keep '42736' value everywhere.

**Comment by geek-soft** [comment]
Events tab is empty :( I would assume that HMCTS Admin can see all events.

screenshot 2018-10-26 at 13 14 07

**Comment by swalker125** [comment]
@geek-soft spoken to Moray. Version history or knowing the events leading up to submitting the case is not important for a hmcts admin. I will update the definition files to display the submitted event only for this user.

**Comment by swalker125** [comment]
screenshot_2018-10-26 core case data hmcts 3
screenshot_2018-10-26 core case data hmcts 4
screenshot_2018-10-26 core case data hmcts 5

File: e2e/pages/login/loginPage.js [hunk 2]
```diff
@@ -14,5 +14,6 @@ module.exports = {
     I.fillField(this.fields.username, username);
     I.fillField(this.fields.password, password);
     I.click(this.submitButton);
+    I.wait(3);
```
Comment (geek-soft) [inline e2e/pages/login/loginPage.js]: Similar wait is already added to `logInAndCreateCase`. If we add wait here we should remove it from actor.

Also we should use `waitForNavigation`, this fixed wait is very fragile.

File: e2e/paths/loginAsHmctsAdmin_test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const config = require('../config.js');
+let caseId;
+
+Feature('Login as hmcts admin');
+
+Before(async (I, caseViewPage) => {
+  I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
+  caseId = await I.grabTextFrom('h2');
+  caseViewPage.goToNewActions(config.applicationActions.submitCase);
+  I.click('.button[type=submit]');
```
Comment (geek-soft) [inline e2e/paths/loginAsHmctsAdmin_test.js]: We should use `/e2e/pages/createCase/addEventSummary.js#submitCase` here with `summary` and `description` set to default `''`

Comment (swalker125) [reply e2e/paths/loginAsHmctsAdmin_test.js]: I don't think I can use this. The test will fail as there is no event description or event summary text fields on the page with the submit button

**Pull request merged**